### PR TITLE
Adds capability for table_footer_view

### PR DESCRIPTION
**Pull request by markrickert**
just like table_header_view so i merged the two constructors into an each loop.

### CODE REVIEW

File: lib/ProMotion/table/table.rb [hunk 1]
```diff
@@ -30,13 +30,15 @@ def promotion_table_data
...
-    def set_up_header_view
-      if self.respond_to?(:table_header_view)
-        header_view = self.table_header_view
-        if header_view.is_a? UIView
-          self.tableView.tableHeaderView = header_view
-        else
-          PM.logger.warn "Table header views must be a UIView."
+    def set_up_header_footer_views
+      [:header, :footer].each do |hf_view|
+        if self.respond_to?("table_#{hf_view}_view".to_sym)
```
Comment (jamonholmgren) [inline lib/ProMotion/table/table.rb]: Don't change this, but just fyi, you can also do:

```ruby
if self.respond_to?(:"table_#{hf_view}_view"))
```

In this case, a string works just as well as a symbol too.

**Comment by jamonholmgren** [comment]
:+1: @ryanlntn, stage for 2.3.

**Comment by ryanlntn** [comment]
Staged for release on the [version-2.3](https://github.com/clearsightstudio/ProMotion/tree/version-2.3) branch.

**Comment by markrickert** [comment]
Documented. https://github.com/clearsightstudio/ProMotion/wiki/API-Reference:-ProMotion::TableScreen

**Comment by jamonholmgren** [comment]
@kevinvangelder Can you verify @markrickert 's docs got into #622?

**Pull request closed without merging**